content/en/docs/languages/java/configuration.md

```diff
@@ -21,37 +25,42 @@ an environment variable:
 - Convert the name to uppercase.
 - Replace all `.` and `-` characters with `_`.
 
-For example `otel.instrumentation.common.default-enabled` would convert to
-`OTEL_INSTRUMENTATION_COMMON_DEFAULT_ENABLED`.
+For example `otel.sdk.enabled` would convert to `OTEL_SDK_ENABLED`.
 
 {{% /alert %}}
 
 ## General
 
-The autoconfigure module registers Java shutdown hooks to shut down the SDK when
-appropriate. Please note that since this project uses `java.util.logging` for
-all of it's logging, some of that logging may be suppressed during shutdown
-hooks. This is a bug in the JDK itself, and not something we can control. If you
-require logging during shutdown hooks, please consider using `System.out` rather
-than a logging framework that might shut itself down in a shutdown hook, thus
-suppressing your log messages. See this
-[JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more details.
-
-{{% alert title="Signal configuration" color="info" %}}
+The
+[autoconfigure module](/docs/languages/java/instrumentation/#automatic-configuration)
+(`opentelemetry-sdk-extension-autoconfigure`) allows you to automatically
+configure the OpenTelemetry SDK based on a standard set of supported environment
+variables and system properties. It is the recommended starting point for
+configuring the SDK.
+
+{{% alert color="info" %}} The autoconfigure module registers Java shutdown
+hooks to shut down the SDK when appropriate. Because OpenTelemetry Java uses
+`java.util.logging` for its logging, some of that logging may be suppressed
+during shutdown hooks. This is a bug in the JDK itself, and not something under
+the control of OpenTelemetry Java. If you require logging during shutdown hooks,
+please consider using `System.out` rather than a logging framework that might
+shut itself down in a shutdown hook, thus suppressing your log messages. See
+this [JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more
+details. {{% /alert %}}
```

Comment:
```suggestion
the control of OpenTelemetry Java. If you require logging during shutdown hooks,
consider using `System.out` rather than a logging framework that might
shut itself down in a shutdown hook, thus suppressing your log messages. See
this [JDK bug](https://bugs.openjdk.java.net/browse/JDK-8161253) for more
details. {{% /alert %}}
```